code/web/src/modules/user/api/actions.js

```diff
@@ -37,7 +39,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, role}', 'token']
+      fields: ['user {id, name, email, role, style}', 'token']
```

Comment:
This adds the id and style so that this will be given back as the response from the API